app/routers/cursor.py

```diff
@@ -0,0 +1,167 @@
+import json
+from pathlib import Path
+from typing import List, Optional, Tuple
+
+from app.dependencies import get_db
+from app.database.models import UserSettings, User
+from app.dependencies import CURSORS_PATH, templates
+from fastapi import APIRouter, Depends, Form, Request
+from sqlalchemy.orm.session import Session
+from starlette.responses import RedirectResponse
+from starlette.status import HTTP_302_FOUND
+
+router = APIRouter(
+    prefix="/cursor",
+    tags=["cursor"],
```

Comment:
I'm not sure we need a route for every setting option. This should probably be routers.setting.py and the calls should be:

* @router.get("/settings/cursor")
* @router.post("/settings/cursor")
* @router.get("/{user_id}/cursor")

Reply:
Did you mean that there shouldn't be a [setting_type].py file for each setting type?
So if I have Audio Settings and Cursor Settings, both of them should be in routers.setting.py, but the @router functions should stay the same, only in this new file instead of the old one?
Thanks for your feedback!